tests/selinux.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+load helpers
+
+@test "selinux test" {
+  if ! which selinuxenabled ; then
+    skip
+   elif ! /usr/sbin/selinuxenabled; then
+     skip
+  fi
+  image=alpine
+  cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json $image)
+  firstlabel=$(buildah --debug=false run $cid cat /proc/1/attr/current)
+  run buildah --debug=false run $cid cat /proc/1/attr/current
+  [ "$status" -eq 0 ]
+  [ "$output" -eq $firstlabel ]
```

Comment:
`test -eq` is used for numeric comparisons; you need `=` here.

Reply:
Fixed

Reply:
Wonder why the tests don't fail.

Reply:
The CI environments are skipping this test.